### ISSUE TITLE
I think there is an error in Day6 SparseVD-solution.ipynb

### ISSUE THREAD

**Issue by lumosity4tpj** [bayesgroup/deepbayes-2019#6]
The activation function of the last layer is F.log_softmax, but the loss is the CrossEntropyLoss.

**Comment by yongduek** [comment]
I just would like to tell you my experience.

As you mentioned, logically speaking, since the last layer is log-softmax, the loss should be NLLLoss.

But practically, you don't have to worry much since there is no harm or not much harm of using CrossEntropyLoss. You can check this through algebraic calculation of log and softmax for a score vector and target label vector.

**Comment by lumosity4tpj** [comment]
Thanks for your experience, I will try.
